### DISCUSSION TITLE
Integer Overflow in procedure TPeDumpChild.SectionListViewData (PeDump.pas)

### DISCUSSION THREAD

**Discussion by ebakkest** [project-jedi/jcl#148] [General]
in procedure TPeDumpChild.SectionListViewData, there are two lines where multiplication by 100 (integer) leads to overflow when your exe is large.

Fix:

Replace 100 with 100.0 in both places to enforce float calculation and avoid the error.

**Comment by obones** [comment]
I'd rather move the `*100` at the end of the line as it's usually what's done in stats software

**Reply by ebakkest** [reply]
Sure, as long as the overflow is fixed :)

**Reply by ebakkest** [reply]
please add the fix?

**Reply by ronaldhoek** [reply]
See [!172](https://github.com/project-jedi/jcl/pull/172)

@ebakkest can you test this?

**Reply by eivindbakkestuen** [reply]
It works for me now, thanks!
